packages/node_modules/@ciscospark/react-component-activity-share-file/src/styles.css

```diff
@@ -32,6 +32,11 @@
   flex: 1 1 auto;
 }
 
+.file-name {
+  max-height: 35px;
+  overflow: hidden;
```

Comment:
Just hiding the overflow isn't to spec. It needs to have ellipsis like this: 
<img width="390" alt="screen shot 2018-01-16 at 2 42 08 pm" src="https://user-images.githubusercontent.com/99547/35008727-71855428-facb-11e7-9cc5-7f88d178ad2d.png">